contrib/scripts/devcontainer-setup.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+### enable debug
+
+set -e
+
+### User Setting
+
+KUBERNETES_VERSION=v1.29
+KIND_VERSION=v0.19.0
```

Comment:
It will be awesome to have renovate bot to update version automatically as well, however, I can check and do it in separate PR.